.github/CODEOWNERS

```diff
@@ -40,7 +40,7 @@
 /core/templates/pages/ @seanlip
 /core/templates/services/ @nithusha21
 /extensions/ @vojtechjelinek
-/scripts/ @ankita240796
```

Comment:
Why was this assigned to me? I only have experience with the E2E tests, not the rest of the scripts

Reply:
Updated.